gdeploy_config/volume_usmqe_gama_disperse_4_plus_2x2.create.conf

```diff
@@ -0,0 +1,24 @@
+#   plus 1 block device as arbiter named (/dev/vdd)
```

Comment:
This comment should be updated to list devices and machines required in the same way as in other gdeploy files.